src/test/regress/output/multi_load_data.source

```diff
@@ -19,3 +19,6 @@ ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 290000;
 \copy lineitem_hash_part FROM '@abs_srcdir@/data/lineitem.2.data' with delimiter '|'
 \copy orders_hash_part FROM '@abs_srcdir@/data/orders.1.data' with delimiter '|'
 \copy orders_hash_part FROM '@abs_srcdir@/data/orders.2.data' with delimiter '|'
+VACUUM ANALYZE;
```

Comment:
Hmm, we don't propagate this to worker nodes. Does this effect the explain output?